specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-11-01-preview/sqlServerAvailabilityGroups.json

```diff
@@ -0,0 +1,802 @@
+            "schema": {
+              "$ref": "#/definitions/SqlServerAvailabilityGroupResource"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***",
+            "schema": {
+              "$ref": "./common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "availability group force failover to this server.": {
+            "$ref": "./examples/ForceFailoverArcSqlServerAvailabilityGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureArcData/sqlServerInstances/{sqlServerInstanceName}/availabilityGroups": {
+      "get": {
+        "tags": [
+          "SqlServerAvailabilityGroups"
+        ],
+        "operationId": "SqlServerAvailabilityGroups_List",
+        "summary": "List the availability group associated with the given Arc Sql Server.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/sqlServerInstanceName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ArcSqlServerAvailabilityGroupListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***",
+            "schema": {
+              "$ref": "./common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets all availability groups associated with an Arc Enabled Sql server.": {
+            "$ref": "./examples/ListBySqlServerInstanceAvailabilityGroup.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AvailabilityGroupConfigure": {
+      "description": "The specifications of the availability group replica configuration",
+      "type": "object",
+      "properties": {
+        "endpointUrl": {
+          "type": "string",
+          "description": "Mirroring endpoint URL of availability group replica"
+        },
+        "availabilityModeDescription": {
+          "type": "string",
+          "description": "The Availability Synchronization mode of the availability group replica."
+        },
+        "failoverModeDescription": {
+          "type": "string",
+          "description": "The failover mode of the availability group replica."
+        },
+        "sessionTimeout": {
```

Comment:
>sessionTimeout [](http://example.com/codeflow?start=9&length=14)

Consider fix to follow naming guidelines:

 YOU SHOULD use a suffix of the unit of measurement for values with a clear unit of measurement (such as bytes, miles, and so on). Use a generally accepted abbreviation for the units (e.g. "Km" rather than "Kilometers") when appropriate.